src/Microsoft.OData.Edm/Vocabularies/Annotations/EdmVocabularyAnnotation.cs

```diff
@@ -44,8 +48,213 @@ public EdmVocabularyAnnotation(IEdmVocabularyAnnotatable target, IEdmTerm term,
             this.term = term;
             this.qualifier = qualifier;
             this.value = value;
+            this.usesDefault = false;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the EdmVocabularyAnnotation class
+        /// to the default value.
+        /// </summary>
+        /// <param name=“target”>Element the annotation applies to.</param>
+        /// <param name=“term”>Term bound by the annotation</param>
+        public EdmVocabularyAnnotation(IEdmVocabularyAnnotatable target, IEdmTerm term)
+        {
+            // Check arguments
+            if (term.DefaultValue == null) // throw error if no default value
+            {
+                // Note: I'm not sure which is the better way of throwing the exception:
+                //throw new ArgumentNullException("value");
+                EdmUtil.CheckArgumentNull(new EdmStringConstant(null), "value");

```

Comment:
CheckArgumentNull is used for parameters passed into the public method.  So, in this case, better to throw an OData exception than cal CheckArgumentNull.

Reply:
Done ✅